sdk/purview/purview-sharing-rest/samples-dev/manageSharesSample.ts

```diff
@@ -0,0 +1,217 @@
+ */
+async function notifyUserSentShareInvitation(
+  client: PurviewSharingClient,
+  sentShareId: string,
+  sentShareInvitationId: string
+) {
+  const result = await client
+    .path(
+      "/sentShares/{sentShareId}/sentShareInvitations/{sentShareInvitationId}:notify",
+      sentShareId,
+      sentShareInvitationId
+    )
+    .post();
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to list sent share invitations
+ *
+ * @summary List sent share invitations
+ */
+async function getAllSentShareInvitations(client: PurviewSharingClient, sentShareId: string) {
+  const initialResponse = await client
+    .path("/sentShares/{sentShareId}/sentShareInvitations", sentShareId)
+    .get();
+
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to list sent shares
+ *
+ * @summary List sent shares
+ */
+async function getAllSentShares(client: PurviewSharingClient, storageAccountResourceId: string) {
+  const options: SentSharesGetAllSentSharesParameters = {
+    queryParameters: {
+      referenceName: storageAccountResourceId,
+    },
+  };
+
+  const initialResponse = await client.path("/sentShares").get(options);
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to get a received share
+ *
+ * @summary Get a received share
+ */
+async function getReceivedShare(client: PurviewSharingClient, receivedShareId: string) {
+  const result = await client.path("/receivedShares/{receivedShareId}", receivedShareId).get();
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to list attached received shares
+ *
+ * @summary List attached received shares
+ */
+async function getAllAttachedReceivedShares(
+  client: PurviewSharingClient,
+  storageAccountResourceId: string
+) {
+  const options: ReceivedSharesGetAllAttachedReceivedSharesParameters = {
+    queryParameters: {
+      referenceName: storageAccountResourceId,
+    },
+  };
+
+  const initialResponse = await client.path("/receivedShares/attached").get(options);
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to delete a received share
+ *
+ * @summary Delete a received share
+ */
+async function deleteReceivedShare(client: PurviewSharingClient, receivedShareId: string) {
+  const initialResponse = await client
+    .path("/receivedShares/{receivedShareId}", receivedShareId)
+    .delete();
+
+  const poller = await getLongRunningPoller(client, initialResponse);
+  const result = await poller.pollUntilDone();
+  console.log(result);
```

Comment:
maybe good to access a property of the result. Is there any interesting there to log?

Reply:
This is the same for all the places where result is being logged. Let's try to log specific info for each schenario